NILICENSE

```diff
@@ -0,0 +1,4 @@
+
+Usage of GrpcSessionOptions on session initialization are subject to the NI General Purpose EULA, available at ni.com
```

Comment:
We already have a LICENSE file. How is NILICENSE and LICENSE different? Why do we need both?

Reply:
Because Legal told me to :)

Reply:
I sent Legal the location of the LICENSE file, and they said we should create a separate file to link to the NI EULA, and not modify the MIT file.